### DISCUSSION TITLE
How to resolve this warning with onedrive.service?

### DISCUSSION THREAD

**Discussion by marcos-data-engineer** [abraunegg/onedrive#2533] [Q&A] [answered]
Hello everyone,

Plis, How to resolve the following warning with onedrive.service:
```
onedrive.service: ProtectHostname=yes is configured, but UTS namespace setup is prohibited (container manager?), ignoring namespace setup
```

Thanks

Marcos

**Comment by abraunegg** [comment]
@marcosvandrade 
Some more information would be helpful - How are you running the client?

* Standalone
* Via Docker
* Via Podman

If via Docker or Podman - did you build the container yourself?

**Reply by marcos-data-engineer** [reply]
@abraunegg I am using UBUNTU 23.10 and running with 'systemctl --user start onedrive'. Maybe bacause a  running as common user? I don´t know, but thanks for feedback. It is working perfectly. I know that is warning only.

**Reply by abraunegg** [reply, marked as answer]
>  I know that is warning only.

Correct. If you want more details however, suggest you ask the systemd developers and/or Ubuntu support regarding that warning message.

**Reply by marcos-data-engineer** [reply]
Ok, thank you.
